### PR TITLE
docs: update readme

### LINKED ISSUE #50: Update Readme

### PR DESCRIPTION
**Pull request by evilrobot-01**
Closes #50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,99 +1,66 @@
...
-| :white_check_mark: `is_in_dispute`                                 |  :white_check_mark:   |       |
-| :white_check_mark: `retrieve_data`                                 |  :white_check_mark:   |       |
+### Dispatchable Functions
 
-### Governance
+#### For Users
+- `add_staking_rewards` - Funds the pallet with staking rewards, which can be used to incentivize oracle usage.
+- `begin_dispute` - Initialises a dispute/vote in the system. Requires a dispute fee to be paid.
+- `fund_feed` - Allows a data feed to be funded with tokens.
+- `send_votes` - Sends any dispute votes due to be sent to the governance controller contract for tallying.
```
Comment (oraclown) [inline README.md:40]: Might want to clarify that it's able to send votes while the voting period hasn't elapsed, and that it doesn't tally the votes.

File: README.md [hunk 2]
```diff
@@ -1,99 +1,66 @@
...
-|----------------------------------------|:------------------:|-------|
-| :white_check_mark: `claim_onetime_tip` | :white_check_mark: |       |
-| :white_check_mark: `claim_tip`         | :white_check_mark: |       |
-| :white_check_mark: `fund_feed`         | :white_check_mark: |       |
-| :white_check_mark: `setup_data_feed`   | :white_check_mark: |       |
-| :white_check_mark: `tip`               | :white_check_mark: |       |
+Anyone can then dispute a reported value, provided they lock a dispute fee. 
+Votes are collated and sent to the Tellor parachain governance contract for tallying and execution.
+A successful dispute results in a disputed reporter being slashed and the slash amount being awarded to the dispute initiator. 
+The dispute fee is awarded to the disputed reporter if the dispute is unsuccessful.
```
Comment (oraclown) [inline README.md:16]: I'd update to specify that fee is awarded to reporter if dispute fails and fee is given back to initiator if tallied and executed as invalid. [src](https://github.com/tellor-io/governance/blob/main/contracts/Governance.sol#L224)

**Review by oraclown — CHANGES_REQUESTED** [review]
Thanks, @evilrobot-01 ! Left some suggestions.

File: README.md (hunk 2 above)
Comment (evilrobot-01) [reply README.md]: Updated to "The dispute fee is awarded to the disputed reporter if the dispute is unsuccessful, but given back to initiator if tallied and executed as an invalid dispute."

File: README.md (hunk 1 above)
Comment (evilrobot-01) [reply README.md]: Updated send_votes to the following:

"Sends any dispute votes to the governance controller contract for tallying, provided the voting period hasn't elapsed."

**Comment by evilrobot-01** [comment]
Rebased

**Review by oraclown — APPROVED** [review]
Thanks, @evilrobot-01 . Looks good

**Pull request merged**